File: csharp/line-up/LineUp.cs
public static class LineUp
{
    public static string Format(string name, int number)
    {
        var lastTwoDigits = number % 100;
        var lastDigit = number % 10;
        string ending;
        if (lastTwoDigits == 11 || lastTwoDigits == 12 || lastTwoDigits == 13) {
            ending = "th";
        } else if (lastDigit == 1) {
            ending = "st";
        } else if (lastDigit == 2) {
            ending = "nd"; 
        } else if (lastDigit == 3) {
            ending = "rd"; 
        } else {
            ending = "th";
        }
        return name + ", you are the " + number + ending + " customer we serve today. Thank you!";
    }
}

File: fsharp/line-up/LineUp.fs
module LineUp

let format (name: string) (number: int): string =
    let ending : string = match number with
                            | i when i % 100 = 11 -> "th"
                            | i when i % 100 = 12 -> "th"
                            | i when i % 100 = 13 -> "th"
                            | i when i % 10 = 1 -> "st"
                            | i when i % 10 = 2 -> "nd"
                            | i when i % 10 = 3 -> "rd"
                            | _ -> "th"
    name + ", you are the " + string(number) + ending + " customer we serve today. Thank you!"
